exams/pearson.py

```diff
@@ -142,6 +143,14 @@ def write(self, tsv_file, rows):
         return (valid_rows, invalid_rows)
 
 
+def split_phone_number(num):
+    """
+    split a phone number into the calling code and the rest of the number
+    """
+    reg = re.compile(r'^\+(\d*)\s?(.*)')
```

Comment:
Several comments here:

* If you're going to compile the regular expression, you're better off doing so outside of the function declaration, so that it can be reused between function calls
* This regular expression could probably benefit from adding a few comments using the [`re.VERBOSE`](https://docs.python.org/2/library/re.html#re.VERBOSE) flag
* The fact that you're using `findall` and then returning the first item in this list makes it difficult to understand what this function is really doing. At first, I thought that this function was misnamed, and was only extracting the calling code from the phone number. Might want to use named captures to make this more clear.
* You're using `*` in several places where you probably want `+` instead. `*` matches 0 or more times, `+` matches 1 or more times. Only `*` if the thing you're matching is optional.

I'd suggest something more like this:

```python
import re

PHONE_NUMBER_RE = re.compile(r"""
    ^    # start of string
    \+   # literal plus sign
    (?P<calling_code>\d+)  # named capture: 1 or more digits
    \s?  # an optional whitespace character
    (?P<number>.+)         # named capture: anything else
""", re.VERBOSE)

def split_phone_number(phone_number):
    """
    Split a phone number into the calling code and the rest of the number
    """
    match = PHONE_NUMBER_RE.search(phone_number)
    if not match:
        msg = "{phone_number} is not a valid phone number".format(
            phone_number=phone_number
        )
        raise ValueError(msg)
    return match.group("calling_code"), match.group("number")
```